kebechet_administrator.py

```diff
@@ -88,7 +92,9 @@ def _handle_solved_message(Configuration):  # noqa: N803
         }
 
         # We store the message to put in the output file here.
-        output_messages.append({"topic_name": "thoth.kebechet-run-url-trigger", "message_contents": message_input})
+        output_messages.append(
+            {"topic_name": KebechetRunUrlTriggerMessage().topic_name, "message_contents": message_input}
```

Comment:
do not put the full topic name. use `KebechetRunUrlTriggerMessage.base_name` the messaging cli will prepend the deployment name when *it* sends the message.

Reply:
Here and elsewhere

Reply:
updated in 0807bf68c29b4228f30acdfdf1efad0816381c9a